template/.github/workflows/preview.yml

```
@@ -0,0 +1,27 @@
+name: Netlify Preview
```

Should we change this file to `deploy-preview.yml` as we're using the deploy action?
```suggestion
name: Deploy Preview
```

---

I've set it to `name: Deploy Preview to Netlify` in order to be consistent with the `Deploy to Netlify` for releases.

✅  ab79900